### DISCUSSION TITLE
Is there any doc for writing a custom generator ?

### DISCUSSION THREAD

**Discussion by Yaojian** [prisma/prisma#5764] [Q&A] [answered]
I want to try something like generating the GraphQL schema from an existing Prisma schema, but cannot find any document on how to write a custom generator ?

**Comment by ryands17** [comment, marked as answer]
@Yaojian 👋 

Unfortunately we do not have any docs currently for writing a custom generator, but you can have a look at these 2 projects created to get a better idea:

https://github.com/pantharshit00/prisma-docs-generator
https://github.com/MichalLytek/typegraphql-prisma

We also have a [docs request](https://github.com/prisma/docs/issues/930) for the same so you can follow this for more updates :)

**Reply by Yaojian** [reply]
thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Reply by fronbasal** [reply]
I feel like custom generators are a core value proposition for using Prisma. Documenting the process of writing custom generators would be very beneficial IMO.
